showkase-annotation/src/main/java/com/airbnb/android/showkase/annotation/ShowkaseComposable.kt

```diff
@@ -52,7 +52,10 @@ package com.airbnb.android.showkase.annotation
  * @param defaultStyle Used to represent a composable function is the default style variant of a given 
  * composable. More information on how Showkase allows you to represent component styles in this section -
  * https://github.com/airbnb/Showkase#representing-component-styles-in-showkase
- *
+ * @param tags Various string values that will be propagated to the Showkase browser to allow additional
+ * filtering or categorization.
+ * @param extraMetadata Various string values that are **not** propagated through the Showkase browser
```

Comment:
Add more information about how it's primarily meant to add more information for custom use cases that you might have